File: Codechef/Practice-Beginner/ALPHABET.cpp
#include<stdio.h>
#include<string.h>

int main() {

	char known[26], word[12];
	short int words;

	scanf("%s %d", known, &words);

	while (words--) {
		scanf("%s", word);
		int i, flag = 0;
		for (i = 0; i < strlen(word); i++) {
			if(strchr(known, word[i]) == NULL) {
				flag = 1;
				break;
			}
		}
		if (flag) printf("%s\n", "No");
		else printf("%s\n", "Yes");
	}

}

File: Codechef/Practice-Beginner/ALPHABET.py
# https://www.codechef.com/problems/ALPHABET

known = list(input())
words = int(input())

for word in range(words):
    flag = False
    for char in input():
        if char not in known:
            print("No")
            flag = True
            break
    if not flag:
        print("Yes")
